demos/serverless/deploy.js

```diff
@@ -221,60 +209,13 @@ function ensureTools() {
   spawnOrFail('npm', ['install']);
 }
 
-function createCaptureS3Buckets(bucketPrefix, regions) {
-  console.log(`Creating S3 buckets for media capture pipelines artifacts.  Bucket prefix: ${bucketPrefix} Regions:[${regions}]`);
-  const lifecycleConfiguration = JSON.stringify({
-    "Rules": [{
-      "ID": "Delete artifacts after 1 day",
-      "Expiration": {"Days": 1},
-      "Status": "Enabled",
-      "Prefix": "",
-    }]
-  });
-  fs.writeFileSync('build/lifecycle_configuration.json', lifecycleConfiguration, {encoding: 'utf8', flag: 'w'});
-  for (bucketRegion of regions) {
-    const bucketName = `${bucketPrefix}-${bucketRegion}`;
-    const bucketPolicy = JSON.stringify({
-      "Id": "Policy1625687208360",
-      "Version": "2012-10-17",
-      "Statement": [{
-        "Sid": "Stmt1625687206729",
-        "Action": [
-          "s3:PutObject",
-          "s3:PutObjectAcl"
-        ],
-        "Effect": "Allow",
-        "Resource": `arn:aws:s3:::${bucketName}/*`,
-        "Principal": {
-          "Service": [
-            chimeMediaPipelinesServicePrincipal
-          ]
-        }
-      }]
-    });
-
-    fs.writeFileSync('build/bucket_policy.json', bucketPolicy, {encoding: 'utf8', flag: 'w'});
-
-    const s3Api = spawnSync('aws', ['s3api', 'head-bucket', '--bucket', `${bucketName}`, '--region', `${bucketRegion}`]);
-    if (s3Api.status !== 0) {
-      if (bucketRegion === 'us-east-1') {
-        spawnOrFail('aws', ['s3api', 'create-bucket', '--bucket', bucketName, '--region', bucketRegion], null, !disablePrintingLogs);
-      } else {
-        spawnOrFail('aws', ['s3api', 'create-bucket', '--bucket', bucketName, '--region', bucketRegion, '--create-bucket-configuration', `LocationConstraint=${bucketRegion}`], null, !disablePrintingLogs);
-      }
-      spawnOrFail('aws', ['s3api', 'put-bucket-policy', '--bucket', bucketName, '--region', bucketRegion, '--policy', 'file://build/bucket_policy.json']);
-      spawnOrFail('aws', ['s3api', 'put-bucket-lifecycle-configuration', '--bucket', bucketName, '--region', bucketRegion, '--lifecycle-configuration', 'file://build/lifecycle_configuration.json']);
-    }
-  }
-}
-
 function copyAssets() {
   fs.copySync('../browser/dist/speech.mp3', 'src/speech.mp3');
   fs.copySync('../browser/dist/speech_stereo.mp3', 'src/speech_stereo.mp3');
 }
 
 function ensureRegion() {
-  if (useChimeSDKMeetings === 'true') {
```

Comment:
The `useChimeSDKMeetings` and similar variables comes from CloudFormation and CloudFormation parameters do not support boolean, thus String was used and compared here.

Reply:
why line 7 initiated as boolean?